src/components/Tool/Appeal/List/AppealTour/AppealTour.test.tsx

```diff
@@ -0,0 +1,146 @@
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+const Components = ({ showTour = true }) => {
+  const appealId = showTour ? ['123', 'list', 'tour'] : ['123', 'list'];
+  return (
+    <ThemeProvider theme={theme}>
+      <TestRouter
+        router={{
+          query: { accountListId, appealId },
+          pathname:
+            '/accountLists/[accountListId]/tools/appeals/[[...appealId]]',
+          isReady: true,
+        }}
+      >
+        <GqlMockedProvider>
+          <AppealsWrapper>
+            <AppealTour />
+          </AppealsWrapper>
+        </GqlMockedProvider>
+      </TestRouter>
+    </ThemeProvider>
+  );
+};
+
+describe('AppealTour', () => {
+  beforeEach(() => {
+    deselectAll.mockClear();
+    toggleSelectAll.mockClear();
+  });
+
+  it('should not render the tour', async () => {
+    const { queryByText, queryByRole } = render(
+      <Components showTour={false} />,
+    );
+    await waitFor(() => {
+      expect(
+        queryByText('Appeal created successfully'),
+      ).not.toBeInTheDocument();
+      expect(queryByRole('button', { name: 'Start' })).not.toBeInTheDocument();
+    });
+  });
+
+  it('should render default view', () => {
+    const { getByRole } = render(<Components />);
+
+    expect(
+      getByRole('heading', { name: 'Appeal created successfully' }),
+    ).toBeInTheDocument();
+    expect(getByRole('button', { name: 'Hide' })).toBeInTheDocument();
+    expect(getByRole('button', { name: 'Start' })).toBeInTheDocument();
+  });
+
+  it('should hide the tour', async () => {
+    const { getByRole, queryByRole } = render(<Components />);
+
+    expect(
+      getByRole('heading', { name: 'Appeal created successfully' }),
+    ).toBeInTheDocument();
+
+    userEvent.click(getByRole('button', { name: 'Hide' }));
+
+    await waitFor(() => {
+      expect(
+        queryByRole('heading', { name: 'Appeal created successfully' }),
+      ).not.toBeInTheDocument();
+      expect(queryByRole('button', { name: 'Start' })).not.toBeInTheDocument();
+    });
+  });
+
+  it('should click through steps of the tour', async () => {
+    const { getByRole, queryByRole, queryByTestId } = render(<Components />);
+
+    await waitFor(() => {
+      expect(
+        getByRole('heading', { name: 'Appeal created successfully' }),
+      ).toBeInTheDocument();
+    });
+
+    userEvent.click(getByRole('button', { name: 'Start' }));
+    expect(
+      getByRole('heading', { name: 'Review Excluded' }),
+    ).toBeInTheDocument();
+    const hideButton = getByRole('button', { name: 'Hide' });
+    const nextButton = getByRole('button', { name: 'Next' });
+    expect(hideButton).toBeInTheDocument();
+    expect(nextButton).toBeInTheDocument();
+
+    userEvent.click(getByRole('button', { name: 'Next' }));
```

Comment:
And all the other times you're clicking the next button.

```suggestion
    userEvent.click(nextButton);
```